main.tf

```diff
@@ -0,0 +1,6 @@
+module "tf_label" {
+  source    = "git::https://github.com/cloudposse/tf_label.git?ref=0.1.0"
```

Comment:
i prefer we just standardize around this format:
```
git::https://github.com/cloudposse/tf_label.git?ref=tags/0.1.0
```

Note: be explicit about `tags/` and use the `.git` suffix.